docs/DECLARING_DEPENDENCIES.md

```diff
@@ -166,14 +166,24 @@ problem, you can import a BOM to force consistent versions, as long as
 you are using at least Gradle 4.6. To do this:
 
 - Turn on BOM support:
-  - If you are using Gradle 4.x, add 
-    `enableFeaturePreview('IMPROVED_POM_SUPPORT')` to `settings.gradle`. 
-   
   - If you are using Gradle 5.x or higher, BOM support is on by default.
+  - If you are using Gradle 4.6 or higher, add 
+    `enableFeaturePreview('IMPROVED_POM_SUPPORT')` to `settings.gradle`.
```

Comment:
Changed the item order to come 5.x come first. Gradle 5.x or higher is more latest (I expect many use 5.x or higher by now) and does not require special statement in settings.gradle.